prebuilts/rabbitmq.toml

```diff
@@ -0,0 +1,71 @@
+#:schema ./schema.json
+# 待审查
+
+id = "rabbitmq"
+name = "RabbitMQ"
+
+# TODO Download svg icon from https://www.svgrepo.com/svg/303576/rabbitmq-logo
+icon = "https://raw.githubusercontent.com/zeabur/service-icons/main/marketplace/rabbitmq.svg"
```

Comment:
Just send a PR to here 😄 It is public → https://github.com/zeabur/service-icons

Reply:
Done